apps/wfo-ui-surf/components/WfoBadges/WfoImpactLevelBadge.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+
+import { TextSize } from '@elastic/eui/src/components/text/text';
+import {
+    WfoBadge,
+    useOrchestratorTheme,
+} from '@orchestrator-ui/orchestrator-ui-components';
+
+import { ImpactLevel } from '../../types';
+
+export type WfoProcessStatusBadgeProps = {
+    impactedObjectImpact: ImpactLevel;
+    size?: TextSize;
```

Comment:
Is textSize related to the possible values in theme.size?

Reply:
not really. It is related to EuiText sizes which are "xs" | "s" | "m" | "relative". 

I could make it like that, pass a theme.size and set it with:

`        <EuiText color={textColor} css={{fontSize: size}}>
            <b>{children}</b>
        </EuiText>
 `
 but I guess I would need a default font size then.
 
The current one is: 
 `
        <EuiText color={textColor} size={size ?? 'xs'}>
            <b>{children}</b>
        </EuiText>
`

Reply:
OR

I can make a new component, something like WfoBigBadge ?
Which will be the same as WfoBadge just with bigger fontSize.